src/main.rs

```diff
@@ -146,7 +146,8 @@ fn create_app<'a>(
                     "boot=<boot_vcpus>,max=<max_vcpus>,\
                     topology=<threads_per_core>:<cores_per_die>:<dies_per_package>:<packages>,\
                     kvm_hyperv=on|off,max_phys_bits=<maximum_number_of_physical_bits>,\
-                    affinity=<list_of_vcpus_with_their_associated_cpuset>",
+                    affinity=<list_of_vcpus_with_their_associated_cpuset>,\
+                    pmu=on|off(only for aarch64)",
```

Comment:
You should make this conditional on the arch.

Reply:
The latest code removed this part